src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -595,8 +596,16 @@ private void handleLogout(@NotNull ByteBuf message) {
 
     private void resetSequence(ByteBuf message) {
         FixField seqNumValue = findField(message, NEW_SEQ_NO_TAG);
+        FixField gapFillMode = findField(message, GAP_FILL_FLAG_TAG);
 
-        if(seqNumValue != null) {
+        if(seqNumValue == null) {
+            LOGGER.trace("Failed to reset servers MsgSeqNum. No such tag in message: {}", message.toString(US_ASCII));
```

Comment:
Maybe use warn instead of trace. Missing MsgSeqNum looks like as serious problem